SConstruct

```diff
@@ -1338,6 +1338,23 @@ env.AlwaysBuild(
             env.PrettyCommand('TIDY', 'src', 'yellow')
         )))
 
+if 'PKG_CONFIG_PATH' in env.Dictionary():
+    pkg_config_path = env['PKG_CONFIG_PATH']
+elif 'PKG_CONFIG' in env.Dictionary() or env.Which('pkg-config'):
+    # https://linux.die.net/man/1/pkg-config the default is libdir/pkgconfig
+    pkg_config_path = os.path.join(env['ROC_SYSTEM_LIBDIR'], 'pkgconfig')
```

Comment:
If `$PKG_CONFIG` or `pkg-config` tool is found, before falling back to `libdir/pkgconfig`, let's first try to ask the tool itself which path it's using:

```
$ pkg-config --variable pc_path pkg-config
/usr/lib64/pkgconfig:/usr/share/pkgconfig
```

If the command works, we can use the first path it prints. If it doesn't, we can fallback to `libdir/pkgconfig`.